src/tekton/bundle-pipeline-task.ts

```diff
@@ -0,0 +1,99 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import { ProgressLocation, ViewColumn, window } from 'vscode';
+import { CliExitData } from '../cli';
+import { Command } from '../cli-command';
+import { checkEnableApiFields, FeatureFlag } from '../debugger/debug';
+import { BundleWizard, TektonType } from '../pipeline/bundle';
+import { K8sTask, TknPipeline, TknPipelineTrigger } from '../tekton';
+import { tkn } from '../tkn';
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as yaml from 'js-yaml';
+import { Platform } from '../util/platform';
+import { getStderrString } from '../util/stderrstring';
+
+interface BundleType {
+  imageDetail: string;
+  resourceDetail: {tektonType: string, name: string}[];
+  userDetail: string;
+  passwordDetail: string;
+}
+
+export async function bundleWizard(): Promise<void> {
+  const featureFlagData: FeatureFlag = await checkEnableApiFields();
+  if (!featureFlagData) return null;
+  if (featureFlagData.data['enable-tekton-oci-bundles'] !== 'true') {
+    window.showWarningMessage('To enable bundles change enable-tekton-oci-bundles to "true" in "feature-flags" ConfigMap namespace tekton-pipelines');
```

Comment:
`tekton-pipelines` should not be hardcoded. This can be for `openshift-pipelines` also.

Reply:
I have removed the hardcoded. Now it will show warning message as per cluster.